### PR TITLE
feat(advanced_app): advanced app documentation

### PR DESCRIPTION
**Pull request by jeromecambon**
Document new advanced app

Relates to [BPM-91](https://bonitasoft.atlassian.net/browse/BPM-91)

[BPM-91]: https://bonitasoft.atlassian.net/browse/BPM-91?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by vhemery — CHANGES_REQUESTED** [review]
The page is builder-declare-interface-in-bonita but navigation still has declare-the-interface-into-bonita link...

File: modules/applications/pages/builder-declare-interface-in-bonita.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+[NOTE]
+====
+Full form edition support will come in a future release.
+====
+
+[NOTE]
+====
+For Subscription editions only.
+====
```
Comment (vhemery) [inline modules/applications/pages/builder-declare-interface-in-bonita.adoc]: Remove this note. Advanced apps are for everyone.

**Review by vhemery — CHANGES_REQUESTED** [review]

File: modules/applications/pages/builder-declare-interface-in-bonita.adoc (hunk 1 above)
Comment (jeromecambon) [reply modules/applications/pages/builder-declare-interface-in-bonita.adoc]: yes indeed...

**Comment by jeromecambon** [comment]
"Contribution check" error is because of the renamed file. This is a worflow issue that has been confirmed by Benjamin.

**Review by vhemery — APPROVED** [review]
LGTM

**Pull request merged**